winter_credit/event.py

```diff
@@ -0,0 +1,49 @@
+"""Class describing an event"""
+
+class Event:
+    """This class describe an event object"""
+
+    def __init__(self, config, date, start, end):
+        self.config = config
+        self.date_dt = date
+        self.start_dt = start
+        self.end_dt = end
+        self.date = date.strftime('%Y-%m-%d')
```

Comment:
We should probably provide a config setting for date format like we do for datetime format.

Reply:
True